runtime/mini-mock/src/hooks.rs

```diff
@@ -94,4 +123,32 @@ impl pallet_clock::traits::OnHookQueues<Runtime> for GlobalOnInitQueues {
 
         total_consumed
     }
+
+    fn process_weekly(_n: BlockNumber, _hook_weight_limit: Weight) -> Weight {
+        0
```

Comment:
Is this unimplemented!()?

Reply:
for now yes, I'm gonna drop queues from attesters and rewards here in next PRs